starwars/public/index.html

```diff
@@ -10,6 +10,8 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <script src="https://unpkg.com/axios/dist/axios.min.js" async></script>
```

Comment:
It's awesome that you looked into axios, but we won't be looking at that as a class for another two weeks haha